apps/web/src/actions/sdk/runDocumentAction.ts

```diff
@@ -28,6 +32,7 @@ export async function runDocumentAction({
   parameters,
 }: RunDocumentActionProps) {
   const { workspace, user } = await getCurrentUserOrError()
+  let tools = []
```

Comment:
Requests to SDK in our playground are stateless. We need to implement a cache to remember where the user stopped because there were one o more tool call requested by the AI